modules/nw-multus-creating-whereabouts-reconciler-daemon-set.adoc

```diff
@@ -51,6 +55,7 @@ spec:
        }
       }
     type: Raw
+   #...
```

Comment:
Personally, I think it would be a good idea to include this type of comment directly before the `spec:` key to imply that there is additional YAML both before and after the `whereabouts-shim` that we're telling the user to include. Does that make sense?